core/src/main/java/org/elasticsearch/cluster/routing/UnassignedInfo.java

```diff
@@ -47,11 +49,20 @@
  */
 public final class UnassignedInfo implements ToXContentFragment, Writeable {
 
+    private static final DeprecationLogger DEPRECATION_LOGGER = new DeprecationLogger(Loggers.getLogger(Setting.class));
```

Comment:
Again, this doesn't need to be tied to the `Setting` logger, we are not deprecating a setting here, only support for certain values. Therefore, I think this should be the logger for `UnassignedInfo.class`.

Reply:
Darn it. Fixed.